core/src/main/java/io/questdb/cairo/RebuildColumnBase.java

```
@@ -281,6 +283,10 @@ private void reindexOneOrAllColumns(
                     );
                 }
             }
+            //To validate if the table has any index or not
+            if (isIndexed) {
```

We definitely don't want to return an error if the table has indexes.
```suggestion
            if (!isIndexed) {
```

---

Will make the changes.